in_toto/in_toto_sign.py

```diff
@@ -207,33 +234,87 @@ def main():
   arguments it then dumps the corresponding file.
   """
   args = parse_args()
-  rsa_key = in_toto.util.import_rsa_key_from_file(args.key)
+  length = len(args.keys)
+  rsa_key = in_toto.util.import_rsa_key_from_file(args.keys[length-1])
+  FILENAME_FORMAT_LAYOUT = "{file_name}.{keyid:.8}.layout"
+  FILENAME_FORMAT_LINK = "{file_name}.{keyid:.8}.link"
```

Comment:
Hm. But isn't it just for the links. I was using it, but then I thought, to make it different for link and layout.